to-do-ui/src/app/components/task-form/task-form.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit } from "@angular/core";
+import { FormBuilder, Validators } from "@angular/forms";
+import { MessageService } from "primeng/api";
+import { DynamicDialogConfig } from "primeng/dynamicdialog";
+
+import { TaskDTO } from "src/app/shared/task-dto";
+import { TaskService } from "src/app/shared/task.service";
+
+@Component({
+	selector: "app-task-form",
+	templateUrl: "./task-form.component.html",
+	styleUrls: ["./task-form.component.css"],
+	providers: [MessageService]
+})
+export class TaskFormComponent implements OnInit {
+
+	public task: TaskDTO = {};
+
+	public constructor(
+		private formBuilder: FormBuilder,
+		private messageService: MessageService,
+		private taskService: TaskService,
+		public config: DynamicDialogConfig,
```

Comment:
```suggestion
		private config: DynamicDialogConfig,
```
make private and declare a global property of actionType

Reply:
this.actionType = this.config.data.actionType;